multi-transfer-esdt/tests/multi_transfer_blackbox_test.rs

```diff
@@ -421,44 +421,44 @@ impl MultiTransferTestState {
             .set_eth_tx_gas_limit(0u64)
             .run();
 
-        self.world
-            .tx()
-            .from(MULTISIG_ADDRESS)
-            .to(ESDT_SAFE_ADDRESS)
-            .typed(esdt_safe_proxy::EsdtSafeProxy)
-            .init_supply_mint_burn(
-                UNIVERSAL_TOKEN_IDENTIFIER,
-                BigUint::from(600_000u64),
-                BigUint::from(0u64),
-            )
-            .run();
-        self.world
-            .tx()
-            .from(MULTISIG_ADDRESS)
-            .to(ESDT_SAFE_ADDRESS)
-            .typed(esdt_safe_proxy::EsdtSafeProxy)
-            .add_token_to_whitelist(
-                TokenIdentifier::from_esdt_bytes("WRAPPED-123456"),
-                "BRIDGE2",
-                true,
-                false,
-                BigUint::zero(),
-                BigUint::zero(),
-                BigUint::zero(),
-                OptionalValue::Some(BigUint::from(ESDT_SAFE_ETH_TX_GAS_LIMIT)),
-            )
-            .run();
-        self.world
-            .tx()
-            .from(MULTISIG_ADDRESS)
-            .to(ESDT_SAFE_ADDRESS)
-            .typed(esdt_safe_proxy::EsdtSafeProxy)
-            .init_supply_mint_burn(
-                WRAPPED_TOKEN_ID,
-                BigUint::from(600_000u64),
-                BigUint::from(0u64),
-            )
-            .run();
+        // self.world
+        //     .tx()
+        //     .from(MULTISIG_ADDRESS)
+        //     .to(ESDT_SAFE_ADDRESS)
+        //     .typed(esdt_safe_proxy::EsdtSafeProxy)
+        //     .init_supply_mint_burn(
+        //         UNIVERSAL_TOKEN_IDENTIFIER,
+        //         BigUint::from(600_000u64),
+        //         BigUint::from(0u64),
+        //     )
+        //     .run();
```

Comment:
should this be deleted if it is not used anymore?

Reply:
Deleted

Reply:
this is still not deleted. Also why this test is not applicable anymore ?

Reply:
Please double check